modules/node-shared/src/test/scala/io/constellationnetwork/node/shared/infrastructure/snapshot/storage/SnapshotLocalFileSystemStorageSuite.scala

```diff
@@ -24,6 +26,9 @@ import SnapshotLocalFileSystemStorage.UnableToPersistSnapshot
 
 object SnapshotLocalFileSystemStorageSuite extends MutableIOSuite with Checkers {
 
+  val hashPathGenerator = PathGenerator.forHash(Depth(2), PrefixSize(3))
```

Comment:
Can we move this number to be in the configuration?

Reply:
as above, however, by explicitly specifying it here too, the test would detect if we would change the configuration by mistake in the snapshot storage class

Reply:
👍🏼